tfjs-backend-webgpu/src/kernels/unary_op_webgpu.ts

```diff
@@ -61,4 +63,22 @@ export class UnaryOpProgram implements WebGPUProgram {
       }
       `;
   }
+
+  getUserCodeWgsl(): string {
+    return `
+      fn unaryOperation(a : f32) -> f32{
+        ${getUnaryOpString(this.op, false, true)}
+      }
+      ${getWorkGroupSizeString(this.workGroupSize)}
+      fn main([[builtin(local_invocation_id)]] localId : vec3<u32>,
```

Comment:
If `localId` is not used, can it be removed?

Reply:
Thanks, this is great, done!